### PR TITLE
Cleanup docstring indentation style

### LINKED ISSUE #300: Fix docstring indentation style
From the style guide, we should be consistently using 4-space indents in docstrings instead of variable ones:
https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments

### PR DESCRIPTION
**Pull request by dthkao**
Fixes #300

---
This change is [Reviewable](https://reviewable.io/reviews/google/mobly/303)

### CODE REVIEW

File: mobly/controllers/android_device_lib/jsonrpc_client_base.py [hunk 1]
```diff
@@ -26,12 +26,12 @@
 {
     "id": <int id of request that this response maps to>,
     "result": <Arbitrary JSON object containing the result of executing the
-               method. If the method could not be executed or returned void,
-               contains 'null'.>,
+        method. If the method could not be executed or returned void,
```
Comment (xpconanfan) [inline mobly/controllers/android_device_lib/jsonrpc_client_base.py]: these are not Args sections, they shouldn't change

**Comment by xpconanfan** [comment]
Review status: 0 of 16 files reviewed at latest revision, 1 unresolved discussion.

---

*[mobly/controllers/android_device_lib/snippet_client.py, line 79 at r1](https://reviewable.io:443/reviews/google/mobly/303#-Kr81WiuMnHE2T6lPe1R:-Kr81WiuMnHE2T6lPe1S:b-25hq6m) ([raw file](https://github.com/google/mobly/blob/68b5602ea81f971d449054b56f9e3f491ce2b09e/mobly/controllers/android_device_lib/snippet_client.py#L79)):*
> ```Python
> 
>         Args:
>             package: (str) The package name of the apk where the snippets are
> ```

inconsistent type statement?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/mobly/303)*

**Comment by dthkao** [comment]
Review status: 0 of 16 files reviewed at latest revision, 2 unresolved discussions.

---

*[mobly/controllers/android_device_lib/jsonrpc_client_base.py, line 29 at r1](https://reviewable.io:443/reviews/google/mobly/303#-Kr8-eMkJSgLB5IhJ1TV-r1-29:-Kr9EiAajH0z9Jx3ObXi:bs635tl) ([raw file](https://github.com/google/mobly/blob/68b5602ea81f971d449054b56f9e3f491ce2b09e/mobly/controllers/android_device_lib/jsonrpc_client_base.py#L29)):*
<details><summary><i>Previously, xpconanfan (Ang Li) wrote…</i></summary><blockquote>

these are not Args sections, they shouldn't change
</blockquote></details>

These are arguably docs on the json args sent to the server and consumed by the client and not an example of actual values, but Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/google/mobly/303)*

**Comment by xpconanfan** [comment]
:lgtm:

---

Review status: 0 of 16 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/google/mobly/303#-:-KrCFy28aA3QZyxn345l:bnfp4nl)*

**Pull request merged**